### PR TITLE
minimal setup

### PR DESCRIPTION
**Pull request by tobi812**

### CODE REVIEW

**Review by tobi812 — COMMENTED** [review]
Also looks good thanks! :)
I think I cannot approve the PR though, because I opened it.

File: herbie/settings.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+APP_LABEL = 'herbieapp'
+
+# Json schema package for validation of business objects
+SCHEMA_REGISTRY_PACKAGE = 'schema'
+
+# chunk size for exporting data
+DEFAULT_CHUNK_SIZE = 100
+
+# message_provider
+MESSAGING_PROVIDER = 'google_pubsub'
```
Comment (tobi812) [inline herbie/settings.py]: Do we still need this variable?

File: herbie/settings.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    'social_core.pipeline.social_auth.social_uid',
+    'social_core.pipeline.social_auth.auth_allowed',
+    'social_core.pipeline.social_auth.social_user',
+    'social_core.pipeline.user.get_username',
+    'social_core.pipeline.user.create_user',
+    'social_core.pipeline.social_auth.associate_user',
+    'social_core.pipeline.social_auth.load_extra_data',
+    'social_core.pipeline.user.user_details',
+    'herbieapp.authentication.social_authentication.process_roles'
+)
```
Comment (tobi812) [inline herbie/settings.py]: I am still not sure if we really want to include the social auth login per default for the admin panel. What do you think about it?

File: schema/customer/customer_v1.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "properties": {
+    "customerId": {
+      "description": "The unique identifier for a customer",
+      "type": "string"
+    },
+    "firstName": {"type": "string"},
+    "lastName": {"type": "string"}
+  },
+  "required": [ "firstName" ]
+}
```
Comment (tobi812) [inline schema/customer/customer_v1.json:16]: Maybe we can move the customer schema related files to a test branch or something like that.

**Review by tobi812 — COMMENTED** [review]
Actually mostly suggestions we should discuss.

File: herbie/settings.py (hunk 1 above)
Comment (ruimbribeiro) [reply herbie/settings.py]: I think we can remove this and the logic assigned to it. Will double check

File: herbie/settings.py (hunk 2 above)
Comment (ruimbribeiro) [reply herbie/settings.py]: Yeah, maybe we can have it as an optional feature instead.

**Pull request merged**